scripts/process-service.js

```diff
@@ -41,6 +42,9 @@ module.exports = (data) => {
     types: types.filter(t => t.category === key),
   })).filter(c => c.types.length > 0);
 
+  const extraImports = unique(
+    types.reduce((acc, t) => acc.concat(t.extraImports || []), []));
+
```

Comment:
here the extra imports are collected and made available to the template context below